app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.html

```diff
@@ -41,6 +41,14 @@ <h4>Manifest Configuration</h4>
         >
         </stage-artifact-selector-react>
       </stage-config-field>
+      <stage-config-field label="Expression Evaluation" ng-if="ctrl.$scope.stage.source === ctrl.artifactSource">
```

Comment:
Do we only want to show the textbox if the manifest comes from an artifact?  I guess that's probably the most common use case, as if it's a text box, you can probably just not type SpEL in the first place...but just wondering if the checkbox should be there for both.

Reply:
That was my initial thought as well, but it seems misleading since as far as I can tell we are [only explicitly evaluating (and failing the stage on the failure to evaluate) SpEL expressions for manifest artifacts](https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestTask.java#L85) in the Deploy Stage.

Reply:
Oh, good catch---I had no idea SpEL evaluation only happened for artifacts.  Makes sense to only show the checkbox in that case then.

Reply:
Confusingly, I think it happens for text manifests as well, during one of the possibly many non-failing passes at expression evaluation that occur when the pipeline begins. We're _explicitly_ evaluating SpEL in artifact manifests just prior to deploy since the full manifest text may only be available _after_ those initial passes.